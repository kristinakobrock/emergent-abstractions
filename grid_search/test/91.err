  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:01, 36.90it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 29.58it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 26.25it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 23.73it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 24.31it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.06it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 23.74it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 23.20it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 23.83it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 23.16it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 23.72it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 24.00it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.72it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 27.41it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 26.81it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.12it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.70it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 27.02it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 30.26it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.59it/s]
