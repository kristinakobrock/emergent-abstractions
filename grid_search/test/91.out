# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7275157570838928, "acc": 0.58339524269104, "length": 3.53756046295166, "mode": "train", "epoch": 1}
{"loss": 0.697548508644104, "acc": 0.49062493443489075, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6855520009994507, "acc": 0.5512378811836243, "length": 3.3588507175445557, "mode": "train", "epoch": 2}
{"loss": 0.6662883758544922, "acc": 0.6201172471046448, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6760022044181824, "acc": 0.6250255107879639, "length": 3.1160073280334473, "mode": "train", "epoch": 3}
{"loss": 0.6964268088340759, "acc": 0.5091796517372131, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6683430671691895, "acc": 0.6136332750320435, "length": 2.6553165912628174, "mode": "train", "epoch": 4}
{"loss": 0.6713927984237671, "acc": 0.6144532561302185, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6641393303871155, "acc": 0.6257761120796204, "length": 2.432175874710083, "mode": "train", "epoch": 5}
{"loss": 0.6692044734954834, "acc": 0.6156249642372131, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.662269651889801, "acc": 0.6251137852668762, "length": 1.9531221389770508, "mode": "train", "epoch": 6}
{"loss": 0.6651148796081543, "acc": 0.6246094107627869, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.660153865814209, "acc": 0.6257320642471313, "length": 1.9686533212661743, "mode": "train", "epoch": 7}
{"loss": 0.6627951860427856, "acc": 0.6259765625, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6590208411216736, "acc": 0.625752866268158, "length": 2.001511573791504, "mode": "train", "epoch": 8}
{"loss": 0.6630820631980896, "acc": 0.6259765625, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6569156646728516, "acc": 0.6258826851844788, "length": 2.060239315032959, "mode": "train", "epoch": 9}
{"loss": 0.6630202531814575, "acc": 0.625781238079071, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6506455540657043, "acc": 0.6295183300971985, "length": 2.4409561157226562, "mode": "train", "epoch": 10}
{"loss": 0.6405409574508667, "acc": 0.6517899036407471, "length": 2.39453125, "mode": "test", "epoch": 10}
{"loss": 0.6341326832771301, "acc": 0.6520957350730896, "length": 3.0417258739471436, "mode": "train", "epoch": 11}
{"loss": 0.6243891716003418, "acc": 0.656890869140625, "length": 3.8984375, "mode": "test", "epoch": 11}
{"loss": 0.6109200716018677, "acc": 0.6760026216506958, "length": 3.471599817276001, "mode": "train", "epoch": 12}
{"loss": 0.6103202700614929, "acc": 0.6517577767372131, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5928031802177429, "acc": 0.6872011423110962, "length": 3.700028896331787, "mode": "train", "epoch": 13}
{"loss": 0.6051380634307861, "acc": 0.6843751668930054, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5753714442253113, "acc": 0.7007393836975098, "length": 3.8187811374664307, "mode": "train", "epoch": 14}
{"loss": 0.6129755973815918, "acc": 0.6886718273162842, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.565123975276947, "acc": 0.7051686644554138, "length": 3.8890819549560547, "mode": "train", "epoch": 15}
{"loss": 0.5734857320785522, "acc": 0.7009766101837158, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5593271255493164, "acc": 0.7075769305229187, "length": 3.92142915725708, "mode": "train", "epoch": 16}
{"loss": 0.5934373140335083, "acc": 0.6925780773162842, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5517892241477966, "acc": 0.7137126922607422, "length": 3.933603286743164, "mode": "train", "epoch": 17}
{"loss": 0.5699753165245056, "acc": 0.7080078125, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5468946099281311, "acc": 0.7171157598495483, "length": 3.9443864822387695, "mode": "train", "epoch": 18}
{"loss": 0.5882425904273987, "acc": 0.6945313811302185, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5395251512527466, "acc": 0.7207475304603577, "length": 3.9499058723449707, "mode": "train", "epoch": 19}
{"loss": 0.5650827884674072, "acc": 0.7039062976837158, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5316504240036011, "acc": 0.7297148108482361, "length": 3.9493117332458496, "mode": "train", "epoch": 20}
{"loss": 0.5744369626045227, "acc": 0.7082032561302185, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5232623815536499, "acc": 0.7351782917976379, "length": 3.9638633728027344, "mode": "train", "epoch": 21}
{"loss": 0.5655446648597717, "acc": 0.7132812142372131, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5153520703315735, "acc": 0.7392377257347107, "length": 3.971493721008301, "mode": "train", "epoch": 22}
{"loss": 0.5584417581558228, "acc": 0.7171874642372131, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5075555443763733, "acc": 0.7479701638221741, "length": 3.978118419647217, "mode": "train", "epoch": 23}
{"loss": 0.5589414238929749, "acc": 0.7267577648162842, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.49685224890708923, "acc": 0.7514841556549072, "length": 3.9817183017730713, "mode": "train", "epoch": 24}
{"loss": 0.5384021401405334, "acc": 0.7240234017372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.48884132504463196, "acc": 0.7569260001182556, "length": 3.9848694801330566, "mode": "train", "epoch": 25}
{"loss": 0.5342949628829956, "acc": 0.7298827767372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.4811289310455322, "acc": 0.7646113634109497, "length": 3.9897472858428955, "mode": "train", "epoch": 26}
{"loss": 0.528372049331665, "acc": 0.7324218153953552, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.47107166051864624, "acc": 0.7705216407775879, "length": 3.989945650100708, "mode": "train", "epoch": 27}
{"loss": 0.5312104225158691, "acc": 0.7388672232627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.46447131037712097, "acc": 0.7782873511314392, "length": 3.9914450645446777, "mode": "train", "epoch": 28}
{"loss": 0.5384799838066101, "acc": 0.7371093034744263, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4587715268135071, "acc": 0.7808495759963989, "length": 3.9896931648254395, "mode": "train", "epoch": 29}
{"loss": 0.5314444303512573, "acc": 0.7443358898162842, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.45277005434036255, "acc": 0.7803412079811096, "length": 3.9935078620910645, "mode": "train", "epoch": 30}
{"loss": 0.5346370935440063, "acc": 0.7484375238418579, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4496675133705139, "acc": 0.7820054888725281, "length": 3.9937546253204346, "mode": "train", "epoch": 31}
{"loss": 0.5312882661819458, "acc": 0.7492188215255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.44331708550453186, "acc": 0.7856675982475281, "length": 3.992443799972534, "mode": "train", "epoch": 32}
{"loss": 0.5281463265419006, "acc": 0.7533202767372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.43802163004875183, "acc": 0.7943649888038635, "length": 3.9932966232299805, "mode": "train", "epoch": 33}
{"loss": 0.5550457239151001, "acc": 0.7414063811302185, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.43117523193359375, "acc": 0.7949150204658508, "length": 3.9927542209625244, "mode": "train", "epoch": 34}
{"loss": 0.5320206880569458, "acc": 0.7531251311302185, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.42567625641822815, "acc": 0.7976025938987732, "length": 3.9913904666900635, "mode": "train", "epoch": 35}
{"loss": 0.5244753360748291, "acc": 0.7578124403953552, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.4204840362071991, "acc": 0.8025088906288147, "length": 3.9925858974456787, "mode": "train", "epoch": 36}
{"loss": 0.5384963750839233, "acc": 0.7513671517372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.4132661819458008, "acc": 0.8043197393417358, "length": 3.9905693531036377, "mode": "train", "epoch": 37}
{"loss": 0.5172117948532104, "acc": 0.7621093392372131, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.4073135256767273, "acc": 0.8101281523704529, "length": 3.991748094558716, "mode": "train", "epoch": 38}
{"loss": 0.5141301155090332, "acc": 0.7601562142372131, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.4013977646827698, "acc": 0.8138355612754822, "length": 3.993656873703003, "mode": "train", "epoch": 39}
{"loss": 0.5320615768432617, "acc": 0.7593750357627869, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.4002220034599304, "acc": 0.8102688193321228, "length": 3.9960248470306396, "mode": "train", "epoch": 40}
{"loss": 0.5218908786773682, "acc": 0.7666015028953552, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.39417561888694763, "acc": 0.8147536516189575, "length": 3.9961068630218506, "mode": "train", "epoch": 41}
{"loss": 0.5206394791603088, "acc": 0.7628907561302185, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.38929399847984314, "acc": 0.8209233283996582, "length": 3.9969730377197266, "mode": "train", "epoch": 42}
{"loss": 0.5216335654258728, "acc": 0.7601562142372131, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3836052119731903, "acc": 0.824898362159729, "length": 3.997208595275879, "mode": "train", "epoch": 43}
{"loss": 0.5151753425598145, "acc": 0.7750000357627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3769198954105377, "acc": 0.8261558413505554, "length": 3.9971821308135986, "mode": "train", "epoch": 44}
{"loss": 0.523622989654541, "acc": 0.7660155892372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.37478405237197876, "acc": 0.8283042311668396, "length": 3.9977293014526367, "mode": "train", "epoch": 45}
{"loss": 0.512093722820282, "acc": 0.7750000357627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.36903852224349976, "acc": 0.8321463465690613, "length": 3.997654676437378, "mode": "train", "epoch": 46}
{"loss": 0.5190521478652954, "acc": 0.7730469107627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.36370086669921875, "acc": 0.8406331539154053, "length": 3.9968440532684326, "mode": "train", "epoch": 47}
{"loss": 0.5040427446365356, "acc": 0.7824218273162842, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.35831019282341003, "acc": 0.8431034684181213, "length": 3.9968674182891846, "mode": "train", "epoch": 48}
{"loss": 0.4928603768348694, "acc": 0.7900391221046448, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3475291132926941, "acc": 0.8497201800346375, "length": 3.997145891189575, "mode": "train", "epoch": 49}
{"loss": 0.48402881622314453, "acc": 0.7937499284744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.34148192405700684, "acc": 0.8540970683097839, "length": 3.9974586963653564, "mode": "train", "epoch": 50}
{"loss": 0.4680842459201813, "acc": 0.8027343153953552, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.33404311537742615, "acc": 0.8613162040710449, "length": 3.998129367828369, "mode": "train", "epoch": 51}
{"loss": 0.4467116594314575, "acc": 0.8101562857627869, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3280273377895355, "acc": 0.8667070865631104, "length": 3.9979257583618164, "mode": "train", "epoch": 52}
{"loss": 0.46670156717300415, "acc": 0.8068360090255737, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3218103349208832, "acc": 0.8703109622001648, "length": 3.998690128326416, "mode": "train", "epoch": 53}
{"loss": 0.4367537498474121, "acc": 0.8216796517372131, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3127627968788147, "acc": 0.8718770146369934, "length": 3.9978880882263184, "mode": "train", "epoch": 54}
{"loss": 0.45288383960723877, "acc": 0.8193358778953552, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.3103765547275543, "acc": 0.8727794289588928, "length": 3.9986939430236816, "mode": "train", "epoch": 55}
{"loss": 0.4459909200668335, "acc": 0.8199219107627869, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3102361857891083, "acc": 0.8729148507118225, "length": 3.9992239475250244, "mode": "train", "epoch": 56}
{"loss": 0.45046675205230713, "acc": 0.8205076456069946, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.3094019591808319, "acc": 0.8735631108283997, "length": 3.999373435974121, "mode": "train", "epoch": 57}
{"loss": 0.43216586112976074, "acc": 0.8232422471046448, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.30475500226020813, "acc": 0.8729871511459351, "length": 3.9993207454681396, "mode": "train", "epoch": 58}
{"loss": 0.42169803380966187, "acc": 0.8189452886581421, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3054906725883484, "acc": 0.8762651085853577, "length": 3.9993443489074707, "mode": "train", "epoch": 59}
{"loss": 0.4406934976577759, "acc": 0.8269530534744263, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.303920716047287, "acc": 0.8757047653198242, "length": 3.998600721359253, "mode": "train", "epoch": 60}
{"loss": 0.44314396381378174, "acc": 0.8273435831069946, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7992857694625854
